lib/deckard/build.ex

```diff
@@ -40,22 +71,22 @@ defmodule Deckard.Build do
     end
   end
 
-  defp build_url(path) do
-    root_url =
-      :deckard
-      |> Application.get_env(__MODULE__)
-      |> Keyword.get(:url, "")
-      |> String.trim()
-      |> String.trim("/")
+  defp scan(channel, cursor \\ 0, matches \\ []) do
+    with ["0", found] <- Redis.query(["SCAN", cursor, "MATCH", "*#{channel}"]) do
+      {:ok, Enum.sort(found ++ matches, &(&1 >= &2))}
+    else
+      [cursor, found] -> scan(channel, cursor, found ++ matches)
```

Comment:
Redis' SCAN needs to be recursively iterated until it gives us "0" as the cursor at which point we have performed a complete scan.